azure.yaml

```
@@ -12,3 +12,13 @@ services:
     host: containerapp
     docker:
       remoteBuild: true
+hooks:
+  predown:
+    windows:
+      shell: pwsh
+      run: ./scripts/pre-down.ps1
+      continueOnError: true
+    posix:
+      shell: sh
+      run: ./scripts/pre-down.sh
+      continueOnError: true
```

```suggestion
      continueOnError: true

```

The pre-commit should fix new lines btw, if you run that.